js/src/personal/maliha/Social.tsx

```diff
@@ -1,56 +1,86 @@
+/* Social Media Icons */
```

Comment:
Usually not a fan of putting the comment above the imports since some ides autocollapse imports and may hide this. Put these comments below the imports.